src/analysis/retail/monk/mistweaver/modules/spells/ManaTea.tsx

```diff
@@ -100,7 +118,7 @@ class ManaTea extends Analyzer {
     }
 
     if (
-      this.selectedCombatant.hasBuff(TALENTS_MONK.MANA_TEA_TALENT.id) &&
+      this.selectedCombatant.hasBuff(SPELLS.MANA_TEA_BUFF.id) &&
```

Comment:
I know it didn't have this before but this rework is the perfect time to make sure we account for other mana reductions in the kit that currently skew this metric - chiji stacks, yulon, CF, TFT vivs etc

Reply:
This might be a good for a normalizer as then it would actually show up in other modules like Cooldowns

Reply:
I agree that it should be a normalizer, I think that this could be applied to every healing spec and you can create a generic extendable ManaCostNormalizer. I am going to limit the scope of this PR and implement a generic version in another PR.